backends/DynamipsBackend.py

```diff
@@ -96,13 +96,13 @@ def genConfig(self, filename):
             filesdir = os.path.dirname(os.path.realpath(sys.modules['__main__'].__file__)) + "/" + path
             if (template['template'] == "bgprouter"):
                 ascfg = open(filesdir + "/bgp.cfg").read()
-                asn =  template['asn']
+                asn = template['asn']
                 neighbors4 = template['neighbors4'].split(';')
                 neighbors6 = template['neighbors6'].split(';')
 
-                for neighbor4 in neighbors4: # 100.64.0.10 as 10;100.64.0.30 as 7;
-                    localas = ascfg;
```

Comment:
lol le ';'...

Reply:
I'm not following you there.

Reply:
J'avais mis des ';' en fin de ligne, drôle d'idée... :)

Reply:
Ok.